docs/theming/developers.md

```diff
@@ -61,21 +156,6 @@ a `<rh-surface>` may be used as a full-width container and provide the
                target="band">
 </uxdot-pattern>
 
-### The `color-palette` attribute
-
-The `color-palette` attribute is a fundamental aspect of the theming system. The 
-attribute is available on specially designated provider elements that actively 
-define a [color palette][palettes], while their children passively accept their 
-background color and text color. The `color-palette` can be set to six possible 
-values:
-
-  * `lightest`
-  * `lighter`
-  * `light`
-  * `dark`
-  * `darker`
-  * `darkest`
-
 ### Theming whole pages
 
 To theme an entire page, you may replace the `<main>` element with an 
```

Comment:
You can do this without adding a `<rh-surface>` if you set your own `color-scheme` rule and include the CSS from `@rhds/tokens/css/color-scheme.css`.  It might win us some points for offering both solutions here.

Reply:
b1768dfc